src/atlas/interpolation/nonlinear/Missing.h

```diff
@@ -29,9 +30,19 @@ struct Missing : NonLinear {
 };
 
 
-template <typename T>
 struct MissingIfAllMissing : Missing {
     bool execute(NonLinear::Matrix& W, const Field& field) const {
```

Comment:
Only missing to move the ::execute methods (MissingIfAllMissing::, MissingIfAnyMissing::, MissingIfHeaviestMissing::execute) to the .cc

Reply:
Good point! Will do.

Reply:
Done now with [b32c316](https://github.com/ecmwf/atlas/pull/176/commits/b32c316e3f31c011ab67688a09bcea7fb3ef016d)